lib/node.js

```diff
@@ -588,7 +588,11 @@ declare class events$EventEmitter {
 
 declare module "events" {
   // TODO: See the comment above the events$EventEmitter declaration
-  declare var EventEmitter: typeof events$EventEmitter;
+  declare class EventEmitter extends events$EventEmitter {
+    static EventEmitter: typeof EventEmitter;
```

Comment:
It's to ensure that `EventEmitter` has a circular dependency of `EventEmitter`. Without it, things like `require('events').EventEmitter` would not work. This mirrors the `EventEmitter` interface in node.

<img width="842" alt="screen shot 2016-08-06 at 8 30 25 am" src="https://cloud.githubusercontent.com/assets/7378772/17456728/27b8557c-5bb0-11e6-9169-1eb16d8c3236.png">